pypika/tests/test_criterions.py

```diff
@@ -763,6 +763,13 @@ def test_with_generator(self):
         self.assertEqual(str(crit), '"a" OR "b" OR "c" OR "d"')
 
 
+class EmptyCriterionTests(unittest.TestCase):
+    def test_field_(self):
```

Comment:
Minor typo in test name. It should be test_fields_ instead.

Reply:
Good catch.